src/NuGetTrends.Data/Program.cs

```diff
@@ -9,13 +9,13 @@ namespace NuGetTrends.Data
 {
```

Comment:
We should convert this project to be a `netcoreapp3.0` class library. Then we don't need all of this, and just have to implement an `IDesignTimeDbContextFactory<NuGetTrendsContext>` to be able to run migrations. Something like this:

```
<PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="3.0.0" />

public sealed class DesignTimeDbContextFactory : IDesignTimeDbContextFactory<NuGetTrendsContext>
    {
        public NuGetTrendsContext CreateDbContext(string[] args)
        {
            // Load	the	settings from the project which	contains the connection	string
            var configuration = new ConfigurationBuilder()
                .SetBasePath(Directory.GetCurrentDirectory())
                .AddJsonFile("appsettings.Development.json", optional: true)
                .Build();

            var builder = new DbContextOptionsBuilder<NuGetTrendsContext>();

            builder.UseNpgsql(configuration["ConnectionStrings:NuGetTrends"]);

            return new NuGetTrendsContext(builder.Options);
        }
    }
```
Not sure we want to do this now.. but we should at some point.

Reply:
Can we do this on a separate PR?
Not sure I follow and I see this could be a separate workload

Reply:
Yes, makes sense. I'll send a PR over.